### ISSUE TITLE
Change scrollbar color/width with CSS override?

### ISSUE THREAD

**Issue by dolanb12** [space10-community/conversational-form#233]
Just a quick question. Is this possible without instantiating with js?

Here's the chatbot link (bottom of page).
[http://qz1.d9c.myftpupload.com/](url)

Having a blast customizing!

Thanks,

Brendan

**Comment by jenssogaard** [comment]
@dolanb12 Yes, I guess you could just override existing CSS for the scrollbar.
